### ISSUE TITLE
InconsistentMigrationHistory when trying to upgrade from 2.18 to 2.19-git (again)

### ISSUE THREAD

**Issue by BhaaLseN** [WeblateOrg/weblate#1860]
### Steps to reproduce
1. attempt to upgrade from 2.18 to 2.19 using Git (`git fetch; git checkout weblate-2.19`)
2. run `./manage.py migrate --no-input` as listed in the upgrade instructions

### Actual behaviour
The migration errors out with this one:
```
django.db.migrations.exceptions.InconsistentMigrationHistory: Migration permissions.0006_auto_20170404_1637 is applied before its dependency lang.0009_auto_20180129_1434 on database 'default'.
```
This is similar to #1838 but not the same message (this one is lang.0009 which was fixed in 0d35e96f2ac9760adfa18050bc51c644629386ab rather than lang.0010 which was supposedly incorrectly added in ed22b1d60c3a6083e11a81f73de717362af77d98 and in turn changed in #1838)

### Expected behaviour
Migration runs as expected

### Server configuration
```
$ ./manage.py list_versions
 * Weblate weblate-2.19
 * Python 3.5.3
 * Django 2.0.2
 * six 1.11.0
 * social-auth-core 1.6.0
 * social-auth-app-django 2.1.0
 * django-appconf 1.0.2
 * Translate Toolkit 2.2.5
 * Whoosh 2.7.4
 * defusedxml 0.5.0
 * Git 2.11.0
 * Pillow (PIL) 1.1.7
 * dateutil 2.6.1
 * lxml 4.1.1
 * django-crispy-forms 1.7.0
 * compressor 2.2
 * djangorestframework 3.7.7
 * user-agents 1.1.0
 * pytz 2018.3
 * pyuca N/A
 * python-bidi 0.4.0
 * PyYAML 3.12
 * Database backends: django.db.backends.mysql
```

This is just me guessing, but going by the file names in the dependencies, I don't think this could ever work (since a 2017 migration lists a 2018 file as dependency?)

**Comment by Pitel** [comment]
Same here.

**Comment by nijel** [comment]
I've just pushed fix for this. It would be great if you could test in your environment. Unfortunately migrations execution seems to depend on filesystem ordering, which can be pretty random...

**Comment by BhaaLseN** [comment]
In fact, I did the same thing as the reporter of #1838 suggested - I commented the two out before running the migration (which is equivalent to 098e9b9e18a53c80a84fbcf92a34ab19f2236a1f); so I'd say this should work. Thanks!

**Comment by nijel** [comment]
Thank you for your report, the issue you have reported has just been fixed. 

* In case you see problem with the fix, please comment on this issue. 
* In case you see similar problem, please open separate issue.
* If you are happy with the outcome, consider supporting Weblate by [donating](https://weblate.org/donate/).

**Comment by nijel** [comment]
Weblate 2.19.1 is out with this fix.

**Comment by Pitel** [comment]
Cinfirming fixed in 2.19.1
